src/css/custom.css

```diff
@@ -16,12 +16,18 @@
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
 
+  /* Light mode colors */
   --ifm-background-color: #ffffff;
   --ifm-background-surface-color: #f2f2f2;
   --ifm-color-content: #1c1e21;
   --ifm-color-content-secondary: #606770;
-}
 
+  /* Brighter link colors for light mode */
+  --ifm-link-color: #0056B3;
+  --ifm-link-hover-color: #003D80;
+  --ifm-link-decoration: underline;
```

Comment:
I am not convinced about the underlines. They work great in the documentation, but I don't like them in the TOCs, menus and sidebar